packages/next-transpile-modules/index.js

```diff
@@ -0,0 +1,41 @@
+module.exports = (nextConfig = {}) => {
+
+  const { transpileModules = [] } = nextConfig
```

Comment:
Using an array as if you namespace things it's easy to have 1, but I can imagine an issue / request coming in to support multiple.